### PR TITLE
EZP-31205: Disabled trigger action from form by hitting enter

### PR DESCRIPTION
**Pull request by lucasOsti**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-31205
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Added preventDefault() for pressed enter key in the form

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/public/js/scripts/admin.content.edit.js [hunk 1]
```diff
@@ -54,6 +54,12 @@
     };
 
     form.setAttribute('novalidate', true);
+    form.onkeypress = (e) => {
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/admin.content.edit.js]:
```suggestion
    form.onkeypress = (event) => {
```

File: src/bundle/Resources/public/js/scripts/admin.content.edit.js [hunk 2]
```diff
@@ -54,6 +54,12 @@
...
+        let key = e.charCode || e.keyCode || 0;
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/admin.content.edit.js]:
```suggestion
        const keyCode = event.charCode || event.keyCode || 0;
```

File: src/bundle/Resources/public/js/scripts/admin.content.edit.js [hunk 3]
```diff
@@ -54,6 +54,12 @@
...
+        if (key == 13) {
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/admin.content.edit.js]:
```suggestion
        if (keyCode === 13) {
```

Please define `13` as a meaningfull variable

File: src/bundle/Resources/public/js/scripts/admin.content.edit.js [hunk 4]
```diff
@@ -54,6 +54,12 @@
...
+            e.preventDefault();
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/admin.content.edit.js]:
```suggestion
            event.preventDefault();
```

**Comment by lucasOsti** [comment]
Outdate PR, current PR for this ticket https://github.com/ezsystems/ezplatform-admin-ui/pull/1164

**Pull request closed without merging**